source/seedsource/tasks/cleanup_tifs.py

```diff
@@ -0,0 +1,20 @@
+from seedsource_project import settings
+import os
+import os.path
+import time
+import re
+from celery.task import task
+
+@task
+def cleanup_temp_tif_files(age=7200):
+    tempDir = settings.NC_TEMPORARY_FILE_LOCATION
+    cutoff = time.time() - age
+    files = os.listdir(tempDir)
+    for file in files:
```

Comment:
`file` is a built-in function name in Python. It would be better to use another name for this variable... `name` perhaps?